### PR TITLE
[MapRouletteUploadCommand] Various changes to maproulette upload command

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This brings in the following changes to the command:
* Replaces the Task name with the _new_ CheckFlag identifiers array.
* Sets the challenge to disabled by default
* Adds a `checkinCommentPrefix` switch to create a  check-in comment prefix `[prefix]: [challenge name]`
* Sets the challenge status to ready

### Potential Impact:

List potential impact to downstream libraries here (ex. atlas-checks, atlas-generator)

### Unit Test Approach:

Uploaded challenges to staging maproulette

### Test Results:

- [x] Challenge disabled
- [x] Check-in comment is: "#maproulette: Anguilla - PoolSizeCheck"
- [x] Status set to 3
- [x] Task title set to new identifiers array

Screen Shot 2020-06-23 at 8 37 18 AM

Screen Shot 2020-06-23 at 8 39 44 AM

### CODE REVIEW

**Comment by danielduhh** [comment]
@dcastrowa good call -- I can update that ask well

File: src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java [hunk 1]
```diff
@@ -189,6 +190,12 @@ private Challenge getChallenge(final String checkName,
         final String challengeName = String.join(" - ", this.getCountryDisplayName(countryCode),
                 result.getName().isEmpty() ? checkName : result.getName());
         result.setName(challengeName);
+        // Add challenge name to check-in comment
+        result.setCheckinComment(String.format("#maproulette: %s", challengeName));
```
Comment (seancoulter) [inline src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java:194]: Do we want the `#` to be configurable? As in referring to a different tag

**Comment by danielduhh** [comment]
Set the updateTask to false by default; see https://staging.maproulette.org/api/v2/challenge/4145
@dcastrowa

**Comment by dcastrowa** [comment]
Nice! Thanks for making these updates.

File: src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java (hunk 1 above)
Comment (danielduhh) [reply src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java]: add a switch for this @seancoulter

Comment (danielduhh) [reply src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java]: added*

**Pull request merged**